src/test/java/kitchenpos/application/CreateMenuHelper.kt

```diff
@@ -0,0 +1,78 @@
+package kitchenpos.application
+
+import kitchenpos.domain.Menu
+import kitchenpos.domain.MenuGroup
+import kitchenpos.domain.MenuGroupRepository
+import kitchenpos.domain.MenuProduct
+import kitchenpos.domain.MenuRepository
+import kitchenpos.domain.Product
+import kitchenpos.domain.ProductRepository
+import org.springframework.stereotype.Component
+import java.math.BigDecimal
+import java.util.*
+
+@Component
+class CreateMenuHelper(
+    private val menuRepository: MenuRepository,
+    private val menuGroupRepository: MenuGroupRepository,
+    private val productRepository: ProductRepository
+) {
+    fun createTestMenu(
+        menuName: String = "메뉴",
+        menuPrice: BigDecimal = BigDecimal("1000"),
+        menuGroupName: String = "메뉴 그룹",
+        isDisplayed: Boolean = false,
+        products: List<ProductDto> = listOf(
+            ProductDto(
+                name = "상품",
+                price = BigDecimal("1000"),
+                quantity = 1L
+            )
+        )
+    ): Menu {
+        val menuGroup = menuGroupRepository.save(
+            MenuGroup().apply {
+                id = UUID.randomUUID()  // ID 할당 추가
+                name = menuGroupName
+            }
+        )
+
+        val savedProducts = products.map { productDto ->
+            if (productDto.id != null) {
+                return@map productRepository.findById(productDto.id).orElseThrow()
+            } else {
+                productRepository.save(
+                    Product().apply {
+                        id = UUID.randomUUID()  // ID 할당 추가
+                        name = productDto.name
+                        price = productDto.price
+                    }
+                )
+            }
```

Comment:
if 문에서 return 이 된다면 else 는 없어도 될 것 같네요 :)
```suggestion
            if (productDto.id != null) {
                return@map productRepository.findById(productDto.id).orElseThrow()
            }
            productRepository.save(
                    Product().apply {
                        id = UUID.randomUUID()  // ID 할당 추가
                        name = productDto.name
                        price = productDto.price
                    }
                )
```